server/go.mod

```diff
@@ -73,7 +73,7 @@ require (
 	golang.org/x/image v0.8.0
 	golang.org/x/net v0.17.0
 	golang.org/x/sync v0.3.0
-	golang.org/x/term v0.13.0
+	golang.org/x/term v0.15.0
```

Comment:
Please revert any changes to go.mod and go.sum.

Reply:
changes have been addressed.